comp/dogstatsd/serverDebug/serverdebugimpl/debug.go

```diff
@@ -29,7 +23,12 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/metrics"
 	"github.com/DataDog/datadog-agent/pkg/tagset"
+	"github.com/DataDog/datadog-agent/pkg/util/defaultpaths"
 	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"github.com/benbjohnson/clock"
+	slog "github.com/cihub/seelog"
+	"go.uber.org/atomic"
+	"go.uber.org/fx"
```

Comment:
I believe the old package order was the preferred one?

Reply:
Yes, not sure why `gopatch` changed it that way. Thanks for catching it